index.html

```diff
@@ -21,12 +21,18 @@
       <div class="hamburger-menu">
         <img src="icons/hamburger-menu.svg" alt="hamburger-menu">
       </div>
+      <ul>
+        <li><a class="nav-links" href="#work">Portfolio</a></li>
+        <li><a class="nav-links" href="#about">about</a></li>
+        <li><a class="nav-links" href="#contact">contact</a></li>
+      </ul>
```

Comment:
- [ ] As one of the requirement is to stick with the figma design, I can see that the margins of nav-links is not as same as in the figma design.Kindly consider fixing it.
- [ ] The exact margin between these nav-links is 32px
your design
![qwer](https://user-images.githubusercontent.com/96683843/213247317-513a0b88-f7fc-4301-96bb-28611ef5b2bc.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/213247299-66748fb5-f1de-4059-b306-f21fa2bc3571.jpg)